setup.py

```diff
@@ -85,14 +85,15 @@
 install_requires = [
     'Flask-BabelEx>=0.9.3',
     'Flask-Breadcrumbs>=0.3.0',
+    'Flask-Login>=0.4.0',
```

Comment:
can you keep the import pinned to same number as in Invenio-Accounts and do:

```python
try:
    from flask_login.utils import _create_identifier
except ImportError:
    from flask_login import _create_identifier
```
instead. Thanks!